build.gradle

```diff
@@ -324,11 +324,12 @@ dependencies {
     compileOnly fg.deobf("com.ldtteam:domum_ornamentum:${domumornamentum_version}:universal")
     compileOnly fg.deobf("com.ldtteam:blockui:${blockui_version}")
     // IMPORTANT. This should be removed/commented when running `runData`
-    runtimeOnly fg.deobf("com.ldtteam:minecolonies:${minecolonies_version}")
+    /*runtimeOnly fg.deobf("com.ldtteam:minecolonies:${minecolonies_version}")
```

Comment:
I pretend I didn't see those

(although I think you should merge from dev/0.8)